snowplow/iglu-client-embedded/schemas/com.metabase/account/jsonschema/1-0-1

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Account creation",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "account",
+    "format": "jsonschema",
+    "version": "1-0-1"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": [
+        "new_user_created",
+        "new_instance_created",
+        "db_connection_banner_click"
+      ],
+      "maxLength": 1024
+    },
+    "version": {
+      "description": "String describing the version of database connection banner we're on",
```

Comment:
@brunobergher I didn't use the description from the doc, since they're not consistent with other descriptions we have. Please let me know if you want to change any descriptions of these `version` or `link` properties.